### ISSUE TITLE
How can we profile 'casacore' and entire CASA using 'gprof'?

### ISSUE THREAD

**Issue by iamharshal** [casacore/casacore#960]
Hi,

First of all, thanks for a wonderful software.

I am interested in profiling CASA using 'gprof'.  However, I could not find any instructions either here or NRAO site on how to achieve it. I gave a try myself -

1. Edited 'cmake' config files, and it generates 'gmon.out' file but its size appears to be too trivial to contain anything. 
2. In addition, discovered the bundled python does not contain debug symbols, and hence 'gprof' could not find entry point.
3. Tried to compile python and to link it with CASA without any success.

Any instructions to achieve this will be highly appreciated.

**Comment by gervandiepen** [comment]
It seems you would like to profile CASA software, but this is casacore (which is used by CASA).

**Comment by iamharshal** [comment]
That's right. A lot of function calls reach here, and I guess any clue here would be relevant for rest of the C++ based components of CASA.
